### ISSUE TITLE
UMD build/example

### ISSUE THREAD

**Issue by kyriacos** [GetStream/stream-chat-react#932]
I tried using:

```
<script src="https://cdn.jsdelivr.net/npm/react@16.13.1/umd/react.production.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/react-dom@16/umd/react-dom.production.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/stream-chat"></script>
<script src="https://cdn.jsdelivr.net/npm/stream-chat-react"></script>
```

Seems stream-chat-react just adds everything on window rather than under an object i.e. StreamChatReact. `stream-chat` is exported under the StreamChat key.

Probably getting something wrong here; was hoping there's a working example of using it directly in the browser. We have a use-case for it.

Tried bundling `stream-chat-react` with our npm package but creating a UMD fails with it as an inline dependency using rollup.

Any suggestions? Thanks!

**Comment by DanC5** [comment]
@kyriacos here are the scripts I have in a current code pen example. The versions are a little outdated but hopefully this will help you out...

```
https://cdn.jsdelivr.net/npm/stream-chat@3.6.2/dist/browser.full-bundle.min.js
https://cdnjs.cloudflare.com/ajax/libs/react/17.0.1/umd/react.production.min.js
https://cdnjs.cloudflare.com/ajax/libs/react-dom/17.0.1/umd/react-dom.production.min.js
https://cdn.jsdelivr.net/npm/stream-chat-react@5.0.3/dist/browser.full-bundle.min.js
```

**Comment by kyriacos** [comment]
Thank you for the links. In fact that's what i am using too. However StreamChatReact adds all its exports on `window`.

For instance when i import `stream-chat` it adds its exports under the StreamChat property i.e. `window.StreamChat`.
Importing `stream-chat-react` just adds everything it exports directly on `window` and not under a property/key. So one would have to for example use `const MessageList = window;` vs `const MessageList = window.StreamChatReact`.

This creates a problem when making a library that requires it. In rollup i have to define our globals as `stream-chat=StreamChat` and `stream-chat-react=window`. StreamChat lib gets this right.

Any chance the stream-chat-react can be updated to export under a key as well?

Also i am using version ~5.

Thanks again!

**Comment by DanC5** [comment]
@kyriacos Thanks for the thorough response, now I follow what you're saying better. I just went into a deep dive on this and see we're writing our exported values to the window in other libraries too. It's not a huge deal to change the name in the full browser bundle export to `StreamChatReact`, I just want to confirm with some of our devs that have been around a while to see if we're writing to the window for a specific reason. If not, then I'll roll out a patch release with the name change early next week.

**Comment by DanC5** [comment]
@kyriacos we just merged a PR that switches our browser bundle from exporting all to window to adding the `StreamChatReact` namespace, https://github.com/GetStream/stream-chat-react/pull/940. Turns out this was a legacy implementation to make setting up Codepens a little easier, but we all agree exporting as a namespace makes more sense now. So thanks for bringing this to our attention and look for a patch release by the end of the week.
